### ISSUE TITLE
Update information on dplyr pages

### ISSUE THREAD

**Issue by simonthelwall** [simonthelwall/simonthelwall.github.io#5]
dplyr information is out of date with deprecated `_each` functions. Need to update to 

```
summarise_at(vars(mpg), funs(sum)) 

mtcars %>% summarise_at(vars(-mpg, - contains("cyl")), funs(sum))
mtcars %>% summarise_at(vars(matches("mpg")), funs(sum))

my_vars <- c("mpg", "cyl")
mtcars %>% summarise_at(vars(my_vars), funs(sum))
```

**Comment by simonthelwall** [comment]
Also add information on crossing and complete

**Comment by simonthelwall** [comment]
Fixed in commit 4edc0b0
